templates/partials/_take-control.html

```diff
@@ -1,12 +1,7 @@
-<section class="p-strip u-no-padding--top">
+<section class="p-strip p-strip--light">
```

Comment:
Just need the one strip class:
```suggestion
<section class="p-strip--light">
```